processes/WIP-bug_bounty_proposal.md

```diff
@@ -0,0 +1,73 @@
+# Proposal to advance a Bug Bounty program for the Node.js® organization
+
+## Introduction
+
+Security is an essential part of the ongoing success of the Node.js community. The community is a vital part of one of the most aggressive markets available providing critical runtime infrastructure for a growing percentage of the internet and corporate computing requirements in 2018. A bug bounty program has become a standard mechanism to improve the security posture of projects, organizations and companies at every level.
```

Comment:
Nit: can you please line break at 80 chars... will make it easier to review. Thank you